generate_openj9_scc.sh

```diff
@@ -278,6 +278,13 @@ function remove_packages() {
     fi
 }
 
+# Changing scc directory permission to 0755
+function change_permissions() {
+    if [ -d "/opt/java/.scc" ]; then
+        chmod -R 0775 /opt/java/.scc
```

Comment:
The comment says `0755`, but the code says `0775`: They can't both be right.
Perhaps it should be `0777`? I think applications need write access and the userids may not been in the same group as `root`.

Reply:
Paging @bharathappali - apologies that was a bad miss by me

Reply:
Ah! Should have thought about it. I was in an impression that application images are built as root and read access should be fine when running as non-root, but yes i feel what @keithc-ca said was right. @karianna  @dinogun I will raise a new PR. Thanks @keithc-ca for the review.